serenata_toolbox/chamber_of_deputies/dataset.py

```diff
@@ -137,3 +132,6 @@ def __translate_file(self, csv_path):
                     encoding='utf-8')
 
         return output_file_path
+
+    def __parse_float(self, string):
```

Comment:
In this case I agree with the change. But feel it should probably come in a new pull request, since this one does not intend to apply this pattern across the repository.

These are the results when searching for the given regex across the project:
`def _[a-z]` -> 5 results found
`def __(?!init)` -> 10 results found